catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/XstreamPathConverter.java

```diff
@@ -63,7 +63,7 @@ public Object unmarshal(HierarchicalStreamReader reader, UnmarshallingContext co
      *
      * @param paths
      */
-    public void setPaths(List<Path> paths) {
+    public void setPaths(LinkedHashSet<Path> paths) {
```

Comment:
Generally I'd wouldn't enforce a particular implementation but it is important that insertion order is preserved.